cmake/projects/Microsoft.GSL/hunter.cmake

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2016-2018, Ruslan Baratov
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_cmake_args)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    Microsoft.GSL
+    VERSION
+    1.0.0
+    URL
+    "https://github.com/ithron/GSL/archive/v1.0.0-h.tar.gz"
```

Comment:
Probably it make sense to wait for https://github.com/Microsoft/GSL/pull/708 to be merged and use https://github.com/Microsoft/GSL ?

Reply:
This is for the 1.0.0 release. So even they merge the PR, the v1.0.0 release won't be updated.

Reply:
As a temporary solution you can put random SHA1 here like:
```cmake
URL
"https://github.com/Microsoft/GSL/archive/5778149583e69b34ab375039ffcc5d51a4dd7b37.zip"
SHA1
"..."
```
so we really need to wait only pull request to be merged.

If it still not what you want then there is a central place for forks: https://github.com/hunter-packages/ I can create `GSL` fork there and give you write permissions.

Reply:
The problem is I'd like the 1.0.0 hunter package to reference to the original 1.0.0 release. 
If I change the URL to point to the PL merge, the hunter package would include some pre-release code, which might not be what users expect.

I think a fork on *hunter-packages* would be a good solution.

Reply:
> the hunter package would include some pre-release code, which might not be what users expect

Branch `hunter-1.0.0` created based on `v1.0.0` tag: https://github.com/hunter-packages/Microsoft.GSL

@ithron I've added you to team (check email), you will be able to push to branches directly but it's better for review reason to create pull request. Please read: https://docs.hunter.sh/en/latest/creating-new/patch.html